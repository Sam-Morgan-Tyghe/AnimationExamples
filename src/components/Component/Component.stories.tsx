import Component from './'

export default {
  component: Component,
  title: 'components/Component',
}

export const base = () => <Component text="Your web app here" />
