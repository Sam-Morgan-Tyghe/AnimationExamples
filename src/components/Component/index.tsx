type ComponentPropTypes = {
  text: string
}

export default function Component({ text }: ComponentPropTypes) {
  return <div data-testid="component">{text}</div>
}
