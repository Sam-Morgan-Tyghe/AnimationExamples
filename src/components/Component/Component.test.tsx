import { render, screen } from '@testing-library/react'

import Component from './'

describe('<Component />', () => {
  const props = {
    text: 'Your new web app here',
  }

  it('Snapshot', () => {
    render(<Component {...props} />)
    expect(screen.getByTestId('component')).toMatchSnapshot()
  })
})
