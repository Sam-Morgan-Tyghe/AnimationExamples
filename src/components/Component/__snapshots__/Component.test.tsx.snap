// Vitest Snapshot v1

exports[`<Component /> > Snapshot 1`] = `
<div
  data-testid="component"
>
  Your new web app here
</div>
`;
